File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/ConferenceEvent.cs
using System;
using ENode.Eventing;

namespace ConferenceManagement
{
    public abstract class ConferenceEvent : DomainEvent<Guid>
    {
        public ConferenceInfo Info { get; private set; }

        public ConferenceEvent() { }
        public ConferenceEvent(ConferenceInfo info)
        {
            Info = info;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/events/ConferenceEvent.java
package com.microsoft.conference.management.domain.events;

import com.microsoft.conference.management.domain.models.ConferenceInfo;
import org.enodeframework.eventing.DomainEvent;

public abstract class ConferenceEvent extends DomainEvent<String> {
    private ConferenceInfo info;

    public ConferenceEvent() {
    }

    public ConferenceEvent(ConferenceInfo info) {
        this.info = info;
    }

    public ConferenceInfo getInfo() {
        return info;
    }

    public void setInfo(ConferenceInfo info) {
        this.info = info;
    }
}
